.github/workflows/build.yml

```diff
@@ -112,7 +134,7 @@ jobs:
             export EXTERNAL_S3_BUCKET_AWS_ACCESS_KEY_ID="${{secrets.EXTERNAL_S3_BUCKET_AWS_ACCESS_KEY_ID}}"
             export EXTERNAL_S3_BUCKET_AWS_SECRET_ACCESS_KEY="${{secrets.EXTERNAL_S3_BUCKET_AWS_SECRET_ACCESS_KEY}}"
 
-            pytest -sv tests/integration
+            pytest -sv tests/integration -n auto --dist loadscope
```

Comment:
Can we add a comment here, e.g.
`use loadscope to avoid issues running tests concurrently that share scoped fixtures`